wolfssl/openssl/evp.h

```diff
@@ -685,6 +685,14 @@ WOLFSSL_API int wolfSSL_PKCS5_PBKDF2_HMAC(const char *pass, int passlen,
                                            const WOLFSSL_EVP_MD *digest,
                                            int keylen, unsigned char *out);
 
+#if defined(HAVE_SCRYPT) && defined(HAVE_PBKDF2) && !defined(NO_PWDBASED) && \
+                                                    !defined(NO_SHA)
+WOLFSSL_API int wolfSSL_EVP_PBE_scrypt(const char *pass, size_t passlen,
+                            const unsigned char *salt, size_t saltlen,
+                            uint64_t N, uint64_t r, uint64_t p, 
+                            uint64_t maxmem, unsigned char *key, size_t keylen);
```

Comment:
`size_t` is fine but let's use `word64` instead of `uint64_t`. Should make porting simpler.

Reply:
It needs to be `size_t` to be compatible with openssl. size_t can be 32-bit or 64-bit depending on architecture.

Reply:
I agree but we should use `word64` instead of `uint64_t`. 

Reply:
Replaced uint64_t with word64. However, I'm worried about changing that signature from openssl. Can we say it is compatible with openssl?

Reply:
Yes let's switch to word64. Most compilers can handle that difference. Our other compatibility functions use word64 like `wolfSSL_OPENSSL_init_ssl`.